src/cssjanus.js

```diff
@@ -6,7 +6,7 @@
  * Copyright 2010 Trevor Parscal
  */
 
-var cssjanus;
+(function() {
```

Comment:
I didn't indent here to try to keep the blame clean. Let me know if you'd like me to indent the contents here anyway.

Reply:
Ended up taking out the IIFE due to linter restrictions.